src/authorization/access-checkers/AgentGroupAccessChecker.ts

```diff
@@ -0,0 +1,77 @@
+import type { Store, Term } from 'n3';
+
+import type { ResourceIdentifier } from '../../ldp/representation/ResourceIdentifier';
+import type { RepresentationConverter } from '../../storage/conversion/RepresentationConverter';
+import type { ExpiringStorage } from '../../storage/keyvalue/ExpiringStorage';
+import { fetchDataset } from '../../util/FetchUtil';
+import { promiseSome } from '../../util/PromiseUtil';
+import { readableToQuads } from '../../util/StreamUtil';
+import { ACL, VCARD } from '../../util/Vocabularies';
+import type { AccessCheckerArgs } from './AccessChecker';
+import { AccessChecker } from './AccessChecker';
+
+/**
+ * Checks if the given WebID belongs to a group that has access.
```

Comment:
Perhaps a bit more around they _why_, explaining that this implements the group functionality of WebACL.